### DISCUSSION TITLE
tg/git-extras are not being installed as expected.

### DISCUSSION THREAD

**Discussion by moonrumble** [zdharma-continuum/zinit#141] [Q&A] [answered]
Hi,

I am installing tj/git-extras using zinit. Below is recommended way to install git-extras. I am still not able to use git commands as expected. 

zinit ice as"program" pick"$ZPFX/bin/git-*" src"etc/git-extras-completion.zsh" make"PREFIX=$ZPFX"
zinit light tj/git-extras

$ZPFX/bin directory is empty. What am I doing wrong?

Thanks

**Comment by moonrumble** [comment]
I had to install column dependencies before it can compile correctly.

**Comment by vladdoster** [comment, marked as answer]
Yes, that would fix it. Additionally, it is documented [here](https://github.com/tj/git-extras/blob/master/Installation.md#dependencies).

**Comment by nurrony** [comment]
@vladdoster @moonrumble I am also facing issues installing `git-extras`. I am using macOS and `column` is already installed on my machine

```bash
❯ which column
/usr/bin/column

```
Here is the snippets
```bash
zi ice as"program" pick"$ZPFX/bin/git-*" make"PREFIX=$ZPFX" nocompile
zi light tj/git-extras
```

My am using **macOS Monterey v12.7.4**

But after cloning `tj/git-extras`, it just stuck in `Building...` state without doing anything

**Comment by nurrony** [comment]
@vladdoster @moonrumble, I got it working as follows
```sh
zi ice as"program" atclone"make install PREFIX=$ZPFX" src'etc/git-extras-completion.zsh' atpull"%atclone" pick"$ZPFX/bin/git-*"
zi light tj/git-extras
```
